src/Apps/Artist/routes.tsx

```diff
@@ -68,7 +68,7 @@ graphql`
 export const routes: RouteConfig[] = [
   {
     path: "/artist/:artistID",
-    Component: ArtistApp,
+    getComponent: () => loadable(() => import("./ArtistApp")),
```

Comment:
For things that we want to split, we implement this interface, which wraps ES6 dynamic `import()` statements.